test/tsconfig.json

```diff
@@ -6,10 +6,9 @@
     "emitDecoratorMetadata": true,
     "sourceMap": true,
     "strictNullChecks": false,
-    "noUnusedLocals": true,
+    "noUnusedLocals": false,
```

Comment:
Also, this was needed to get TS 2.6.x to not complain about some of our test classes that have decorators on them, but are otherwise "unused."

Reply:
Can you explain this a little bit more or provide a link? Is this a bug since 2.6?